### ISSUE TITLE
Confusion Matrix Dialog has wrong caption for validation set

### ISSUE THREAD

**Issue by bhrnjica** [bhrnjica/anndotnet#2]
Confusion matrix dialog window has wrong title. It stand "Confusion matrix for Training data set", event if the confusion matrix is opened for validation data set.

**Comment by bhrnjica** [comment]
fixed.
